crime-applications-adaptor/src/main/resources/schemas/crimeapplicationsadaptor/common/assessmentDetail.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "assessmentDetail.json",
+  "type": "object",
+  "description": "Assessment details",
+  "properties": {
+    "applicantAmount": {
+      "type": "number",
+      "description": "An explanation about the purpose of this instance."
+    },
+    "applicantFrequency": {
+      "type": "string",
+      "description": "The Frequency of Applicant",
+      "enum": ["WEEKLY", "2WEEKLY", "4WEEKLY", "MONTHLY", "ANNUALLY"]
+    },
+    "assessmentDetailCode": {
+      "type": "string",
+      "description": "Assessment Detail Code"
+    }
```

Comment:
Should these all be required fields? Will there ever be the case that there is an AssessmentDetails object that is empty?